Android/android.py

```diff
@@ -573,8 +580,9 @@ def parse_args():
     test = subcommands.add_parser(
         "test", help="Run the test suite")
     test.add_argument(
-        "-v", "--verbose", action="store_true",
-        help="Show Gradle output, and non-Python logcat messages")
+        "-v", "--verbose", action="count", default=0,
+        help="Show Gradle output, and non-Python logcat messages. "
+        "Use twice to include high-volume messages which are rarely useful.")
```

Comment:
Once this is merged, I'll switch the the buildbot over to `-v` mode so it will capture all non-Python log messages except the high-volume ones from test_syslog. These messages have been critical for understanding test failures in the past, including SELinux errors, low memory warnings, and signals being delivered to the wrong thread.

Since `-v` also displays the Gradle build log, there will no longer be any need for the buildbot to have a separate `build_testbed` step, hence the change to its comment above.